Sources/Sentry/SentryProfiler.mm

```diff
@@ -306,126 +349,97 @@ + (void)startWithHub:(SentryHub *)hub
         return;
     }
 
-#    if SENTRY_HAS_UIKIT
-    [SentryDependencyContainer.sharedInstance.framesTracker resetProfilingTimestamps];
-#    endif // SENTRY_HAS_UIKIT
```

Comment:
This is removed from here to SentryTracerConcurrency, where we can know how many profilers are being held waiting for their associated tracers to finish. Only when there are no profilers left do we reset this data.